### ISSUE TITLE
Clicking "Finder" does not refresh or recenter map

### ISSUE THREAD

**Issue by andrewviren** [CityOfPhiladelphia/parks-rec-finder#109]
[Spec. Ref.  v1.0.4](https://drive.google.com/file/d/15CBYAYFlovs7OOkizQJogSDNMTYamAAJ/view)  | URL | App Version | Browser (version) - Size | Device
---------- | --- | ----------- | ------------------------ | ------
pg13.1  |#/activities|  0.1.0 | Chrome - 1024 x 768 |  Desktop

### Expected Behavior

After I search for a zipcode and change the map zoom or focus, I expect the map to refresh and recenter itself when I click on Finder link in upper left of page.

###Observed Behavior

If I search a zipcode (19143), zoom out (or zoom into a particular location or locations) and then click the finder link at the top, the category list view reappears, but the map does not recenter and keeps whatever view I was last on. This can be frustrating if you are zoomed too far in or out and want to recenter the map. 

> Steps to Reproduce

1. Opened finder
2. Typed "19143" into zip code search and hit enter.
3. Zoom out too far, or zoom or pan to any part of the map. Or choose any program or location to zoom into that location
4. Clicked Finder Link in upper left corner.

#### Proposed Solution
As a user, if I scrolled too far from my search results or zoomed too far in or out, I would want to be able to start over from a zoomed out / centered map by clicking "Finder".

![clickingfinderlinkdoesnotrefreshmap](https://user-images.githubusercontent.com/719422/35303456-ffe8113a-005f-11e8-8c7e-94002516298e.PNG)

![image](https://user-images.githubusercontent.com/719422/35303515-342d5e6e-0060-11e8-884a-832cd94b2a4a.png)

**Comment by timwis** [comment]
Should be quick fix. `src/components/pprf-header.vue` L34

```html
        <router-link class="app-title" to="/">
            <h2 class="text-nopad">FINDER</h2>
        </router-link>
```
Just change it from `<router-link>` to `<a href"/">`

Sara's working on that file right now so don't want to create merge conflict.

**Comment by karissademi** [comment]
@timwis Using an <a> tag will reload the page. Does that seem like expected behavior?

**Comment by timwis** [comment]
> After I search for a zipcode and change the map zoom or focus, I expect the map to refresh and recenter itself when I click on Finder link in upper left of page.

I think so, yes. Clicking the app name should take you to the start, essentially resetting your experience. Is there an alternative?

**Comment by karissademi** [comment]
I'm not sure if there was another way to center on the map without a page reload. This seems perfectly fine, just wanted to note it.

**Comment by karissademi** [comment]
Closed in #130
